src/components/Header/Banner.js

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useState } from 'react'
+import { Button, GU, IconCross, textStyle, useTheme } from '@aragon/ui'
+import background from '../../assets/banner.svg'
+
+function Banner() {
```

Comment:
Dont you wanna keep all the assets together outside `components` ? In case they are needed in more than one component, so we dont have to duplicate them or look for them in other folders

Reply:
Was thinking that co-locating assets that are specific to a component would be easier, but this makes sense too!